static/styles/left_sidebar.css

```diff
@@ -71,6 +71,10 @@ li.show-more-topics {
     float: right;
```

Comment:
![image](https://user-images.githubusercontent.com/25124304/122146727-2c05b880-ce75-11eb-9d4e-9250116b9008.png)

Reply:
This commit message should mention `streams_inline_cog` instead, since that's what this hides, right?